test/e2e/app_management_test.go

```diff
@@ -545,7 +545,7 @@ func TestConfigMap(t *testing.T) {
 }
 
 func TestFailedConversion(t *testing.T) {
-
+	t.Skip()
 	defer func() {
 		FailOnErr(Run("", "kubectl", "delete", "apiservice", "v1beta1.metrics.k8s.io"))
 	}()
```

Comment:
@alexmt - This test is failing on recent versions of K3s, so I disabled it. I am not sure what use-case this test actually covers, can we remove it or do we need to update it?

Reply:
The test was created to verify edge case when known k8s type cannot be converted. I think it is better to replace it with unit test in https://github.com/argoproj/gitops-engine repo, unless it already exist. 

I'm ok to remove it. I think e2e test is overkill in this case.

Reply:
I think it's due to the fact that K3s with more recent versions ship with the `v1beta1.metrics.k8s.io` APIService already enabled, as opposed to i.e. the microk8s I use for local development, and possibly the very old version of K3s we use for e2e tests currently. 

The end-to-end test creates the this APIService and expects it to sync - in recent K3s versions, it won't sync and is healthy from the beginning tho.

I will keep the `t.Skip()` inside of an envvar check for `ARGOCD_E2E_K3S` instead of removing the test.

Reply:
sounds good!